codec_specs.md

```diff
@@ -763,3 +783,34 @@ Codec Name: VobBtn Buttons
 
 Description: Based on [MPEG/VOB PCI packets](http://dvd.sourceforge.net/dvdinfo/pci_pkt.html). The file contains a header consisting of the string "butonDVD" followed by the width and height in pixels (16 bits integer each) and 4 reserved bytes. The rest is full [PCI packets](http://dvd.sourceforge.net/dvdinfo/pci_pkt.html).
 
+## Private Data Extension Blocks
+
+When indicated in the codec description as possible, `Private Data` codec specific data MAY be followed by one or more extension blocks.
+
+Each extension block begins with a 4-byte extension block size field which is the size of the extension block minus 4 (excluding the size of the extension block size field), followed by an extension block identifier field; extension block identifier field MAY be identical to the ones found in ISO/IEC 14496-12 and its extensions.
+
+Registered `Private Data` extension block identifier fields are:
```

Comment:
Each codec should probably tell which indentifier are OK. For example `hvcE` shouldn't be legal in H264.

Reply:
I limited `hvcE` and `mvcC` extensions to a specific `Codec ID`.